src/Components/Publishing/Layouts/__tests__/NewsLayoutAdsEnabled.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { DisplayCanvas } from "Components/Publishing/Display/Canvas"
+import { NewDisplayCanvas } from "Components/Publishing/Display/NewDisplayCanvas"
+import { NewsArticle } from "Components/Publishing/Fixtures/Articles"
+
+import { mount } from "enzyme"
+import "jest-styled-components"
+import React from "react"
+import renderer from "react-test-renderer"
+import { NewsLayout } from "../NewsLayout"
+jest.mock("isomorphic-fetch")
+
+declare const global: any
+global.fetch = jest.fn(() =>
+  Promise.resolve({
+    status: 200,
+    json: () => Promise.resolve({}),
+  })
+)
+
+// tslint:disable: jsx-boolean-value
+describe("News Layout with new ads enabled", () => {
+  const getWrapper = () => {
+    return mount(
+      <NewsLayout
+        areHostedAdsEnabled={true}
+        article={NewsArticle}
+        shouldAdRender={true}
```

Comment:
Like @damassi said you can just do `shouldAdRender` instead of `shouldAdRender={true}`